### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard :"Toggle More " menu item is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5717]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: MacOS Mojave 10.14.5

**Additional Detail:**
Tool Name: Keyboard
MAS Reference: MAS2.1.1

**Steps to Reproduce :**
1.	Install the app Azure Data Studio.
2.	Connect to the Database->Go to file->New Notebook.
3.	Select query->Click on the Run command->click on text tab
4.	Try to navigate “Toggle More ” menu item using keyboard.

**Actual:**
“Toggle More ” menu item is not accessible through keyboard.

**Similar issue is observed for the below pages:**
**1.NotebookQueryTable:** Navigate to the connected server->Select the database->Navigate to the  file->New Notebook->Select any query->Click on the Run command

**Expected:** 
“Toggle More ” menu item should be accessible through keyboard.

**User Impact:**
User with disabilities find it difficult to navigate to “Toggle More” item list.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
MAS2.1.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[5717_[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3232211/5717_.Accessibility.A11y_AzureDataStudio_TextTab__Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
This is a dupe of #5719.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Verified the bug in
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5
and still it is repro, hence reopened the bug.
[5717_[Accessibility]A11y_AzureDataStudio_TextTab__Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3285260/5717_.Accessibility.A11y_AzureDataStudio_TextTab__Keyboard.pptx)
